tests/unit/Spanner/Session/CacheSessionPoolTest.php

```diff
@@ -17,15 +17,14 @@
 
 namespace Google\Cloud\Tests\Unit\Spanner\Session;
 
-require_once __DIR__ . '../../../Core/Lock/MockGlobals.php';
+require_once __DIR__ . '/../../../../src/Core/Testing/Lock/MockGlobals.php';
```

Comment:
This is the only part that I know for sure will not survive the split into the subrepo. I don't know what the best approach to refactoring the MockGlobals file is to fix this.